editor/src/components/canvas/controls/select-mode/padding-resize-step-indicator.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import { canvasPoint, CanvasPoint, CanvasVector } from '../../../../core/shared/math-utils'
+import { assertNever } from '../../../../core/shared/utils'
+import { useColorTheme } from '../../../../uuiui'
+import { controlForStrategyMemoized } from '../../canvas-strategies/canvas-strategy-types'
+import { EdgePiece } from '../../canvas-types'
+import { deltaFromEdge } from '../../padding-utils'
+import { CanvasOffsetWrapper } from '../canvas-offset-wrapper'
+
+export const PaddingResizeStepIndicatorTestId = 'PaddingResizeStepIndicatorTestId'
+
+export interface PaddingResizeStepIndicatorProps {
+  currentPaddingValue: number
+  activeEdge: EdgePiece
+  dragDelta: CanvasVector
+  dragStart: CanvasPoint
+}
+
+export const PaddingResizeStepIndicator =
```

Comment:
I think this needs to be renamed, as the word `step` in here implies that it is an indicator for a discrete value that the user can jump to, whereas this indicator is for showing the current padding value

Reply:
done!